platforms/js/build_js.py

```diff
@@ -233,6 +235,7 @@ def build_loader(self):
     parser.add_argument('--emscripten_dir', default=emscripten_dir, help="Path to Emscripten to use for build (deprecated in favor of 'emcmake' launcher)")
     parser.add_argument('--build_wasm', action="store_true", help="Build OpenCV.js in WebAssembly format")
     parser.add_argument('--disable_wasm', action="store_true", help="Build OpenCV.js in Asm.js format")
+    parser.add_argument('--disable_single_file', action="store_true", help="Do not merge JavaScript and WebAssembly into one single file")
```

Comment:
I think it make sense highlight wasm in the flag naming. Something like `--disable_wasm_merge` or `--build_standalone_wasm`.

Reply:
However, origin option is `SINGLE_FILE` so it's also logical to keep `--disable_single_file` :thinking:

Reply:
Agree, pros and cons, no strong opinion from my side on argument name :slightly_smiling_face: If we are to move away from `single_file` in the argument name I think I personally like your second suggestion the best, i.e. `--build_standalone_wasm` (we then avoid the word `disable`, which maybe makes the argument name slightly easier to interpret by removing a negation).

Happy to modify the PR if you decide on some other name instead of `--disable_single_file`.